### PR TITLE
Location time section

### LINKED ISSUE #124: Location + Date on main site
- [ ] Add in the date / location according to this design (mobile, desktop, day, night mode) 
- [ ] Sarah didn't update the night mobile version but it's the same as the day mobile except with the desktop night colors 

Screenshot 2023-01-13 at 2 51 58 PM
Screenshot 2023-01-13 at 2 52 19 PM
Screenshot 2023-01-13 at 2 52 31 PM

### PR DESCRIPTION
**Pull request by srazh**
Added location and date section to main site
Fixes #124.

Changelist:

- added location and date section to main site
- responsive for mobile and changes colors based on day/night mode 

Notes:

Tested:
- localhost3000

Screenshots & Screencasts:
Screen Shot 2023-01-18 at 8 32 16 PM
Screen Shot 2023-01-18 at 8 32 33 PM

Screen Shot 2023-01-18 at 8 31 54 PM
Screen Shot 2023-01-18 at 8 32 05 PM

### CODE REVIEW

File: main-site/components/landing-section/LandingSection.styles.ts [hunk 1]
```diff
@@ -40,12 +40,12 @@ const StyledStar = styled(motion.img)`
...
    {
     color: ${colors.WHITE};
-    padding-bottom: 0.5em;
+    padding-bottom: 0.1em;
     @media ${max.tablet} {
       display: flex;
       justify-content: center;
       align-items: center;
-      padding-bottom: 1em;
+      padding-bottom: 0.2em;
```
Comment (jzhzhang66) [inline main-site/components/landing-section/LandingSection.styles.ts]: can be 1 liner: padding: 0.2 em 0;

File: main-site/components/location-time-section/LocationDateSection.styles.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+import { max } from '../../../shared-ui/lib/responsive';
+import { StyledLocationSectionProps } from '../../../shared-ui/lib/types';
+import { colors } from '../../../shared-ui/style/colors';
+import { H1, P } from '../../../shared-ui/style/typography';
+
+const StyledLocationDateSectionContainer = styled.div`
+    padding-bottom: 1.5em;
+
+
```
Comment (jzhzhang66) [inline main-site/components/location-time-section/LocationDateSection.styles.ts]: remove extra spaces

File: main-site/components/location-time-section/LocationDateSection.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { LocationSectionProps } from "../../../shared-ui/lib/types";
+import { StyledDateHeader, StyledLocationDateSectionContainer, StyledLocationText } from "./LocationDateSection.styles";
+
+
+const LocationTimeSection: React.FC<LocationSectionProps> = ({ isDay }) => {
+    return (
+
```
Comment (jzhzhang66) [inline main-site/components/location-time-section/LocationDateSection.tsx]: remove extra new line

**Review by jzhzhang66 — APPROVED** [review]
LGTM!!!! just some small cleanup comments before you merge

File: main-site/components/location-time-section/LocationDateSection.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    return (
+       <StyledLocationDateSectionContainer>
+        <StyledDateHeader isDay={isDay}>February 10-12, 2023</StyledDateHeader>
+        <StyledLocationText isDay={isDay}>@Wood Mackenzie (Formerly known as PowerAdvocate)</StyledLocationText>
+        <StyledLocationText isDay={isDay}>179 Lincoln St, Boston, MA 02111</StyledLocationText>
+       </StyledLocationDateSectionContainer>
+    );
+}
+
+export default LocationTimeSection;
```
Comment (deanframe) [inline main-site/components/location-time-section/LocationDateSection.tsx:15]: nit add newline

File: shared-ui/lib/types.ts [hunk 5]
```diff
@@ -99,3 +99,11 @@ export interface MobileTeamColumnProps {
...
   isClickable: boolean;
 }
+
+export interface StyledLocationSectionProps {
+  isDay: boolean;
+}
+
+export interface LocationSectionProps {
+  isDay: boolean;
+}
```
Comment (deanframe) [inline shared-ui/lib/types.ts:109]: newline

**Review by deanframe — APPROVED** [review]
LGTM with a couple of dumb formatting changes. 

Can you also run `yarn` in the root directory and in the `main-site` directory? I'm wondering why there's a 347 line diff in `yarn.lock`

**Pull request merged**